File: practica2/Circulo.java
import java.awt.*;

public class Circulo {
    Punto centro;
    int radio;

    public Circulo(Punto centro, int radio) {
        this.centro = centro;
        this.radio = radio;
    }

    public void dibujaCirculo(Graphics g) {
        g.drawOval(centro.x - radio, centro.y - radio, 2 * radio, 2 * radio);
    }
}

File: practica2/circulo.py
import turtle
class Circulo:
    def __init__(self, centro, radio):
        self.centro = centro
        self.radio = radio
    def __str__(self):
        return f"Círculo con centro en {self.centro} y radio {self.radio}"

    def dibujarCirculo(self):
        turtle.penup()
        turtle.goto(self.centro.x * 20, (self.centro.y - self.radio) * 20)
        turtle.pendown()
        turtle.circle(self.radio * 20)
        turtle.penup()
